jenkins/Jenkinsfile.nightly

```diff
@@ -47,68 +41,34 @@ pipeline {
 	options {
 		timeout(time: 10, unit: 'MINUTES')
 	}
+	environment {
+		ENABLE_HTTP_API='http-version3,http-status'
+		ENABLE_WS_API='rpc-v3,blockchain'
+		LISK_APP_WS='ws://host.docker.internal:7887'
+		ENABLE_TESTING_MODE='true'
+    }
 	stages {
 		stage('Checkout SCM') {
 			when {
 				anyOf {
 					branch 'development'
-					branch 'feature/**'
+					branch 'release/**'
 				}
 			}
 			steps {
 				script { echoBanner(STAGE_NAME) }
 				cleanWs()
-				dir('lisk-core') {
-					checkout([$class: 'GitSCM', branches: [[name: "${params.COMMITISH_CORE}" ]], userRemoteConfigs: [[url: 'https://github.com/LiskHQ/lisk-core']]])
-				}
```

Comment:
Would it be possible to clone the development branch of Lisk Core, build the docker image and then proceed with the following steps?

Reply:
Yes, it is possible but require more efforts. As we have core beta.0 image available, we can update this later in separate issue, what do you think?